.github/workflows/tests.yml

```diff
@@ -209,6 +209,7 @@ jobs:
       LXD_VERBOSE: "1"
       LXD_OFFLINE: "1"
       LXD_TMPFS: "1"
+      GOTRACEBACK: "crash"
```

Comment:
So this will prevent Go based panics from displaying a stack trace in the log and we will need to the manually inspect the core dump files? Or will it do both?

Reply:
On my local environment I can still see everything in logs and in addition to that I'm getting a coredump.
Without this parameter I coredumps are not generated. `go version go1.23.3 linux/amd64`

Reply:
I found a description of the different settings here: https://dave.cheney.net/tag/gotraceback

With `GOTRACEBACK=crash` we should get everything outputted and additionally a core dump